File: question-three/weather-forecast.kt
fun main() {
    addCity("Nairobi", 25.0, 60.0, 15.0)
    addCity("Mombasa", 30.0, 70.0, 20.0)
    println(weatherData)
}

val weatherData = mutableMapOf<String, Map<String, Any>>()
fun addCity(city: String, temperature: Double, humidity: Double, windSpeed: Double): Map<String, Map<String, Any>> {
    weatherData[city] = mapOf(
        "temperature" to temperature,
        "humidity" to humidity,
        "wind_speed" to windSpeed
    )
    return weatherData
}

File: question-three/weather-forecast.js
const weatherData = {};
function addCity(city, temperature, humidity, windSpeed) {
    weatherData[city] = {
        temperature: temperature,
        humidity: humidity,
        wind_speed: windSpeed
    };
    return weatherData;
}

addCity("Nairobi", 25, 60, 15);
addCity("Mombasa", 30, 70, 20);
console.log(weatherData);